README.md

```diff
@@ -28,7 +28,7 @@ libraryDependencies += "ch.epfl.scala" %% "tasty-query" % "<latest-version>"
 You can find the latest release in the Releases list on GitHub.
 
 Head over to the [latest API docs](https://javadoc.io/doc/ch.epfl.scala/tasty-query_3/latest/tastyquery.html) to see what's available.
-To get started, create a [`Classpath`](https://javadoc.io/doc/ch.epfl.scala/tasty-query_3/latest/tastyquery/Classpaths$$Classpath.html) using [`ClasspathLoaders.read`](https://javadoc.io/doc/ch.epfl.scala/tasty-query_3/latest/tastyquery/jdk/ClasspathLoaders$.html).
+To get started, create a [`Classpath`](https://javadoc.io/doc/ch.epfl.scala/tasty-query_3/latest/tastyquery/Classpaths$$Classpath.html) using [`ClasspathLoaders.read`](https://javadoc.io/doc/ch.epfl.scala/tasty-query_3/latest/tastyquery/jdk/ClasspathLoaders$.html). Please note that TASTy Query requires all classes, such as the JRE, must be explicitly added to the classpath. 
```

Comment:
We keep one sentence per line, to make diffs easier:
```suggestion
To get started, create a [`Classpath`](https://javadoc.io/doc/ch.epfl.scala/tasty-query_3/latest/tastyquery/Classpaths$$Classpath.html) using [`ClasspathLoaders.read`](https://javadoc.io/doc/ch.epfl.scala/tasty-query_3/latest/tastyquery/jdk/ClasspathLoaders$.html).
Please note that TASTy Query requires all classes, such as the JRE, must be explicitly added to the classpath.
```

Reply:
Perhaps also add a note that on JVM >= 9, we can use `FileSystems.getFileSystem(java.net.URI.create("jrt:/")).getPath("modules", "java.base")` for that?

Reply:
I added this and a notice about JS